jupyterhub/alembic/env.py

```diff
@@ -56,14 +65,14 @@ def run_migrations_offline():
 
     """
     connectable = config.attributes.get('connection', None)
+    config_opts = dict(literal_binds=True)
+    config_opts.update(common_config_opts)
```

Comment:
Feel free to ignore. It doesn't matter, but it made me need to think "hmmm is common_config_ops including literal_binds? If so, that could lead to a bug.

```suggestion
    config_opts = {}
    config_opts.update(common_config_opts)
    config_opts["literal_binds"] = True
```